src/cascadia/TerminalCore/TerminalDispatch.hpp

```diff
@@ -16,6 +16,10 @@ class TerminalDispatch : public Microsoft::Console::VirtualTerminal::TermDispatc
     bool SetGraphicsRendition(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions* const rgOptions,
                               const size_t cOptions) override;
 
+    bool PushGraphicsRendition(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::GraphicsOptions* const rgOptions,

```

Comment:
@zadjii-msft, if we don't have a todo to move this adapter next to the other one and deduplicate some of its work through some sort of interface, we should do so. Do you know of an existing item like that, or should we make one?

Reply:
This is now #3849